Helper_Scripts/ci/check_imports_and_methods.py

```
@@ -117,7 +121,11 @@ def run_pytest(pytest_args: Iterable[str]) -> int:
     print("Running:", " ".join(cmd))
     print("PYTHONDONTWRITEBYTECODE=", env["PYTHONDONTWRITEBYTECODE"])
 
-    proc = subprocess.run(cmd, cwd=str(REPO_ROOT), env=env)
+    proc = subprocess.run(  # nosec B603 - fixed local argv, no shell or untrusted command composition
```

Ruff (0.15.6)

[error] `subprocess` call: check for execution of untrusted input

(S603)